.github/workflows/spread.yml

```diff
@@ -45,15 +45,13 @@ jobs:
     - name: Set up Spread
       run: |
         set -euo pipefail 
-        curl -s -O https://people.canonical.com/~chrishr/spread.snap
+        sudo snap install spread-mir-ci
         sudo snap run lxd init --auto
```

Comment:
There seem to be some GitHub actions we could use instead of setting up LXD ourselves. Maybe something to look into later.